src/middlewared/middlewared/plugins/kubernetes_linux/utils.py

```diff
@@ -1,4 +1,5 @@
 BACKUP_NAME_PREFIX = 'ix-applications-backup-'
 KUBECONFIG_FILE = '/etc/rancher/k3s/k3s.yaml'
+KUBERNETES_WORKER_NODE_PASSWORD = 'e3d26cefbdf2f81eff5181e68a02372f'
```

Comment:
Same password for everybody? Doesn't this open possibility for a local exploit?

Reply:
We are going to disallow any more nodes from joining the k3s cluster in another PR ( this is not happening right now ), which means that we won't have more nodes joining, so this does not matter ?